src/EA.Weee.Web/Areas/AATFReturn/ViewModels/SelectReportOptionsNilViewModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace EA.Weee.Web.Areas.AatfReturn.ViewModels
+{
+    using System;
+    using System.Globalization;
+
+    public class SelectReportOptionsNilViewModel
```

Comment:
Possible to inherit this model from the ReturnViewModelBase is it has many of these properties available?

Reply:
Yep, I've made changes so it inherits